### ISSUE TITLE
2FA and update pages look weird - box is too wide

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/server#7161]
Ref https://github.com/nextcloud/server/pull/6736#issuecomment-343945869

![bildschirmfoto von 2017-11-13 15-58-12](https://user-images.githubusercontent.com/1374172/32732522-2957112a-c88d-11e7-8623-093b3ad777e1.png)
![bildschirmfoto von 2017-11-13 15-45-26](https://user-images.githubusercontent.com/1374172/32732523-297a8236-c88d-11e7-8b60-6c91a5282285.png)

**Comment by jancborchardt** [comment]
@Espina2 could you look into this? :)

**Comment by Espina2** [comment]
@jancborchardt Should I do a mock or code? :p

**Comment by jancborchardt** [comment]
Code please, as your pull request #6736 broke the style from before which was as intended, and should be restored. :)

**Comment by jancborchardt** [comment]
While fixing this I found another issue: The logo and text vertically moves when there are processes like updater or reset password which change the height of the content. This looks very strange and makes things feel unstable.

Anyone has an idea how to fix that? Otherwise I would propose to revert the change since vertically centering all the time will inherently cause the issue I described. cc @nextcloud/designers
